src/testing/recording.ts

```diff
@@ -75,6 +77,7 @@ export function setupRecording({
   }
 
   return new Polly(name, {
+    ...options,
```

Comment:
How would you feel about doing using [this](https://lodash.com/docs/4.17.15#defaultsDeep) lodash function and allowing deep properties to be specified? I have a need to overwrite the `matchRequestsBy` values and will probably end up implementing it anyways.

Reply:
We talked offline but for the sake of documenting, I'm working on this implementation.